test/integration/simple_example/controls/client.rb

```diff
@@ -31,6 +32,21 @@
     its('exit_status') { should eq 0 }
   end
 
+  describe command('forseti inventory list') do
+    its('exit_status') { should eq 0 }
+  end
+
+  describe command('pip show forseti-security|grep Version') do
+    its('exit_status') { should eq 0 }
+
+    let(:formated_stdout) do
```

Comment:
This can be expressed as `its('stdout.chomp') { should match "Version: #{forseti_version[1..-1]}" }`

Also, why are we trimming the `forseti_version` string?

Reply:
`var.forseti_verison` has `v` infront. Example `v2.13.0`. I didn't trim it in the output

Reply:
If that's the case, let's indicate that in code. Using the subscript operator hides this detail, but we can do something like this:

```ruby
let(:forseti_version) { forseti_version.sub(/\Av/, '') }
```

That way the code directly expresses the fact that we're dropping the 'v' prefix.